### DISCUSSION TITLE
Frigate is friggin' awesome

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#3294] [General]
I've been using Frigate for about a week and I absolutely love it. I've been looking for something exactly like this for a long time. The closest I got was shinobi, but it was too convoluted and I never got things running smoothly no matter how much fiddling I did. DW Spectrum was pretty good but also had weird UI paradigms, no object detection, no birds-eye, phoned home, and most critically, had no reasonably priced licenses for smaller deployments. Blue Iris is windows-only, consumed a lot of resources, and I'm not sure how well it would do as a server (I didn't try it long). At one point I began writing my own nvr but didn't find enough time to get too far with it. Frigate should be more widely considered the class leader. 

I wanted to give a big thanks to the team for all the great work they do!

I'm also a software engineer who specializes in front-end and would love to contribute to polishing the web interface. I'll start another thread next week to find out how to get started if that's something the project could use.

All the best!

**Comment by jvrobert** [comment]
Yeah, I agree - Blake's done an amazing job, and Nick and all the other contributors have really started stepping up. Between Frigate, Double-take, and Compreface we can do things just out of the box that I couldn't even have imagined just a few years ago.
